### ISSUE TITLE
`exp-lookit-video` videos not playing in Firefox?

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/ember-lookit-frameplayer#415]
## TL;DR

The videos presented in a `exp-lookit-video` frame may not be playing in Firefox.

## How to reproduce

Using the EFP JSON protocol below, the video does not play in Firefox. There are no console errors, and there are no event logs about starting the video - the event logs stop at the "next frame" event, which is the last event from the previous frame when it transitions to the `exp-lookit-video` frame.

```json
"video": {
  "kind": "exp-lookit-video",
  "baseDir": "https://raw.githubusercontent.com/UW-LIDL/lookit-stimuli-template/master/",
  "videoTypes": [
     "mp4"
  ],
  "video": {
    "source": "boing"
  }
}
```

The same protocol above does work in Chrome.

## Testing environment details

- Production site
- EFP version: afb2fda97f9325476da70e2bc0e4602816011c60 (2024-10-04)
- Firefox 127.0.1
- Chrome 129.0.6668.90

**Comment by becky-gilbert** [comment]
This was fixed by updating Firefox (133.0).
